packages/url/src/index.js

```diff
@@ -34,6 +34,51 @@ export function addQueryArgs( url, args ) {
 	return baseUrl + '?' + stringify( { ...query, ...args } );
 }
 
+/**
+ * Returns a single query argument of the url
+ *
+ * @param  {string} url URL
+ * @param  {string} arg Query arg name
+ *
+ * @return {Array|string} Query arg value.
+ */
+export function getQueryArg( url, arg ) {
+	const queryStringIndex = url.indexOf( '?' );
```

Comment:
Or just use a proper URL parsers, like `require( 'url' ).parse( url ).search`†

https://nodejs.org/docs/latest/api/url.html#url_url_search

† I assume reason we haven't is due to size of the imported (stub) module.